dashboard/src/app/workspaces/workspace-details/environments/machine-config/edit-machine-name-dialog/edit-machine-name-dialog.controller.ts

```diff
@@ -17,30 +18,42 @@
  * @author Oleksii Kurinnyi
  */
 export class EditMachineNameDialogController {
+  private name: string;
+  private origName: string;
+  private machinesNames: string[];
+  private callbackController: WorkspaceMachineConfigController;
 
   /**
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor($mdDialog) {
+  constructor(private $mdDialog: ng.material.IDialogService) {
```

Comment:
it's a way to create and initialize a member in one place
see **Parameter properties** in [Classes](http://www.typescriptlang.org/docs/handbook/classes.html)